plugins/Jobs.cpp

```diff
@@ -1209,6 +1209,14 @@ bool BedrockPlugin_Jobs::processCommand(SQLite& db, BedrockCommand& command) {
             if (!db.writeIdempotent(updateQuery)) {
                 STHROW("502 RequeueJobs update failed");
             }
+
+            // If we want to update the name, let's do that
+            const string& name = request["name"];
```

Comment:
Why not try to make this update in the query above instead of doing 2 separate queries?

Reply:
IMO what Ionatan is suggesting should be a blocker before we start making use of this new option. Or are there any downsides to doing it in the same query?

Reply:
So for this I tried to follow how the structure was written elsewhere in this file and it seems like we run multiple update commands depending if the param was passed. Another example of how name was updated is [here](https://github.com/Expensify/Bedrock/blob/e857107be743af28490fc109af6d2144223b6939/plugins/Jobs.cpp#L994). 

Do you feel like having it combined is definitely better? If so i can update this. Not sure why it would be a blocker though but whatever you guys feel.

Reply:
I don't think this is a blocker, but will let Florian decide that.
As a general rule, we should always try to make commands do the least amount of queries we can... I have no idea why we did 2 in that example you pasted, maybe an oversight or maybe it made the code harder to read/understand (as it is only applying to one command), not sure really.

Reply:
Cool sounds good. I'll combine the queries and keep in mind that for future we'd prefer fewer queries over here :+1:

Reply:
@fnwbr mind if i update the code in a follow up PR so we don't block the Web-E PR on this.

Reply:
Heh, well, my point would be do update this _before_ we start making use of this.

But I can't judge about the performance. Maybe the current average load is so small that we wouldn't even notice if it's 2 queries or only 1.

I'd say: Let's not hold Web, but prioritize getting the follow-up Bedrock PR out there.